src/styles/styles.js

```diff
@@ -29,7 +29,7 @@ const picker = {
     paddingHorizontal: 11,
     paddingBottom: 8,
     paddingTop: 23,
-    height: 50,
+    height: variables.inputHeight,
```

Comment:
I can find any of the variables that you changed in this file. Am I going crazy?

Reply:
I think they might have been removed in one of those times when you merged with main

Reply:
I think they are added on this PR in `variables.js`, unless I'm missing something.

Reply:
Oh you are right, I was looking to a specific commit and couldn't  find them, like I said, going insane hehe